src/components/GetStarted/Faqs.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+// @ts-ignore
+import { GU, Link } from '@aragon/ui'
+import { fontWeight } from '../../style/font'
+import LayoutLimiter from '../Layout/LayoutLimiter'
+import Faq from './Faq'
+import PageHeading from '../PageHeading/PageHeading'
+
+// TODO: Add url when available
+const FAQS_URL = ''
+
+function Faqs(): JSX.Element {
+  return (
+    <LayoutLimiter size="medium">
+      <div
+        css={`
+          magin-top: 100px;
+          text-align: center;
```

Comment:
I think this is being applied at the wrong level as it causes all of the faq buttons to be centered as well.